docs-go/flows.md

```diff
@@ -0,0 +1,187 @@
+# Flows
+
+Flows are functions with some additional characteristics: they are strongly
+typed, streamable, locally and remotely callable, and fully observable.
+Firebase Genkit provides CLI and developer UI tooling for working with flows
+(running, debugging).
+
+## Defining flows
+
+A flow wraps a function:
+
+- {Go}
+
+  ```go
+  menuSuggestionFlow := genkit.DefineFlow(
+    "menuSuggestionFlow",
+    func(ctx context.Context, restaurantTheme string, _ genkit.NoStream) (string, error) {
+      suggestion := makeMenuItemSuggestion(restaurantTheme)
+      return suggestion, nil
+    },
+  )
+  ```
+
+You can define input and output schemas for a flow, which is useful when you
```

Comment:
This suggests that people will in fact define input and output schemas for a flow, but that's not what they do.  What they do is write a flow that accepts or returns a structured input value.  Maybe something like (assuming this is accurate):

The argument and result types of a flow may be structured values. Genkit will produce JSON schemas for these values using invopop/jsonschema. Those JSON schemas will be used by the Genkit CLI and UI.